.github/workflows/cross-mac.yml

```diff
@@ -38,4 +39,24 @@ jobs:
         with:
           name: ${{ matrix.target }}
           path: ./artifacts/
-          retention-days: 3
\ No newline at end of file
+          retention-days: 3
+
+  bb_wrapper:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: cachix/install-nix-action@v18
+        with:
+          nix_path: nixpkgs=channel:nixos-22.11
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build barretenberg_wrapper
+        working-directory: ./barretenberg_wrapper
+        # This removes the lockfile as a workaround to relative file paths causing a failure in flakes
```

Comment:
can you say more about this @phated

Reply:
@kobyhallx there seems to be this issue: https://github.com/NixOS/nix/issues/3978#issuecomment-952418478

Nix relies on the cache SHA to be exactly the same for relative paths to work, but for some reason it is not the same between computers (e.g. when I cloned from you or when I pushed to this CI). 

This problem will go away when we have the barretenberg flake it the upstream repository since we can rely on the github url.